frontend/.vscode/extensions.json

```
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+       // Fill in any extensions
```

```suggestion
       "bradlc.vscode-tailwindcss"
```

Eigentlich nur das... @world-wide-weights/frontend Habt ihr noch andere Plugins in use welche sehr gut sind?

---

added